src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -1,376 +1,167 @@
-                        $('#current_currency_crypto').text(crypto_currencies);
-                    }
-
-                    currencies.forEach((currency) => {
-                        let is_allowed = true;
-                        other_currencies.forEach((other_currency) => {
-                            if (currency === other_currency) {
-                                is_allowed = false;
-                            }
-                        });
-                        if (is_allowed) {
-                            if (isCryptocurrency(currency)) {
-                                has_all_crypto = false;
-                                $crypto_2.find('ul').append(`<li>${Currency.getCurrencyFullName(currency)}</li>`);
-                            } else {
-                                $fiat_1.find('ul').append(`<li>${Currency.getCurrencyFullName(currency)}</li>`);
-                            }
-                        }
-                    });
-
-                    if (has_all_crypto) {
-                        $crypto_2.setVisibility(0);
-                    }
-
-                }
+        reason_checkbox_list = document.getElementsByName('reason-checkbox');
+        el_dialog_container = getElementById('dialog_container');
+        el_form_closure_step_1 = getElementById('form_closure_step_1');
+        el_step_2_back = getElementById('step_2_back');
+        el_step_2_submit = getElementById('step_2_submit');
+        el_other_trading_platforms = getElementById('other_trading_platforms');
+        el_suggested_improves = getElementById('suggested_improves');
+        el_remain_characters = getElementById('remain_characters');
+        el_account_closure_warning = getElementById('account_closure_warning');
+        el_account_closure_error = getElementById('account_closure_error');
+        el_closure_loading = getElementById('closure_loading');
+        el_deacivate_button = getElementById('deactivate');
+        el_error_msg = getElementById('error_msg');
+        el_submit_loading = getElementById('submit_loading');
+
+        el_closure_loading.setVisibility(1);
+        const hideDialogs = () => {
+            el_account_closure_warning.setVisibility(0);
+            el_account_closure_error.setVisibility(0);
+        };
+        hideDialogs();
 
+        const is_virtual = !hasAccountType('real');
+        BinarySocket.wait('landing_company').then(() => {
+            if (!is_virtual) {
                 BinarySocket.send({ statement: 1, limit: 1 });
                 BinarySocket.wait('landing_company', 'get_account_status', 'statement').then(async () => {
                     const is_eligible = await Metatrader.isEligible();
                     if (is_eligible) {
-                        $('.metatrader-link').setVisibility(1);
+                        const MT5_links = document.getElementsByClassName('metatrader-link');
+                        Array.from(MT5_links).forEach(MT5_link => { MT5_link.setVisibility(1); });
```

Comment:
you could use the `applyToAllElements` as well by passing `'.metatrader-link'` instead of manually doing these two lines